git-hub

```diff
@@ -580,7 +560,10 @@ def cli():
 @click.option('--label', '-l', default="", help="search PR by label")
 def hub(command, args, user, comment, number, branch, sort, opened, label):
     if command == "pr":
-        pr_num = int(args[0])
+        if args[0].isdigit():
```

Comment:
What is this code doing?

Reply:
It checks if the argument supplied is an integer (so either the pr number(i.e. 4) or the url(e.g. https://github.com/machine-shop/oss_devkit/pull/35)) so that it knows whether to take it as an integer or to find the pr number inside the url.

Reply:
It changed the url I inputted into my comment. But like if the user inputted instead of '30', they inputted 'https://github.com/machine-shop/oss_devkit/pull/ 30'

Reply:
OK, I suggest adding comments to both branches of the "if", saying what you're expecting in each branch.  This will help future developers see what you're doing.

Reply:
added comments.